templates/front/my_checks_desktop.html

```diff
@@ -49,11 +52,21 @@
                 copy
             </button>
         </td>
+
+        <td class="nag-cell">
+            {% if check.nag_mode == 1 %}
+                <span class="status icon-up"></span>
+            {% elif check.nag_mode == 0 %}
+                <span class="status icon-up new"></span>
+            {% endif %}
+        </td>
+
         <td class="timeout-cell">
             <span
                 data-url="{% url 'hc-update-timeout' check.code %}"
                 data-timeout="{{ check.timeout.total_seconds }}"
                 data-grace="{{ check.grace.total_seconds }}"
+                data-nag="{{ check.nag_interval.total_seconds }}"
```

Comment:
👍

Reply:
One more thing; the nag mode icon on the checks does not seem to toggle the mode when clicked. Is that how its supposed to work?

Reply:
To rephrase; when is the nag mode turned on?

Reply:
@collinmutembei It's turned on when the check goes down and turned off when the check is resolved

Reply:
@briankabiro All this happens automagically, right?

Reply:
Yeah, automatically.

Reply:
🤔 Doesn't seem to work for me

<img width="1211" alt="screen shot 2017-09-14 at 18 06 17" src="https://user-images.githubusercontent.com/10899666/30437564-97247c32-9977-11e7-9591-e7b3ccf5a1f5.png">

![screen shot 2017-09-14 at 18 06 36](https://user-images.githubusercontent.com/10899666/30437568-9bb97572-9977-11e7-8bec-b2b4431ad382.png)